cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/validator/ApacheValidator.java

```diff
@@ -0,0 +1,389 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.template.etl.validator;
+
+import co.cask.cdap.api.annotation.Description;
+import co.cask.cdap.api.annotation.Name;
+import co.cask.cdap.api.annotation.Plugin;
+import co.cask.cdap.template.etl.api.Validator;
+import org.apache.commons.validator.GenericValidator;
+import org.apache.commons.validator.routines.CreditCardValidator;
+import org.apache.commons.validator.routines.DateValidator;
+import org.apache.commons.validator.routines.DomainValidator;
+import org.apache.commons.validator.routines.DoubleValidator;
+import org.apache.commons.validator.routines.EmailValidator;
+import org.apache.commons.validator.routines.FloatValidator;
+import org.apache.commons.validator.routines.ISBNValidator;
+import org.apache.commons.validator.routines.InetAddressValidator;
+import org.apache.commons.validator.routines.IntegerValidator;
+import org.apache.commons.validator.routines.LongValidator;
+import org.apache.commons.validator.routines.RegexValidator;
+import org.apache.commons.validator.routines.ShortValidator;
+import org.apache.commons.validator.routines.UrlValidator;
+
+/**
+ * Utility class containing validator functions provided by org.apache.commons.validator.routines package.
+ */
+@Plugin(type = "validator")
+@Name("apache")
+@Description("Apache validator functions from org.apache.commons.validator.routines package. " +
+  "Includes isDate(String), isCreditCard(String), isBlankOrNull(String), isEmail(String), " +
+  "isInRange (value, min, max)[supported types : double, int, float, short]," +
+  "isInt(String), isLong(String), isShort(String), isUrl(String), matchRegex(String pattern, String input)," +
```

Comment:
This should not list all the methods that are supported. We should add this to documentation. Saying about what it does would be good enough. Can you move this to documentation page please within a table.